README.md

```diff
@@ -59,6 +70,15 @@ kubectl delete namespace cattle-resources-system
 
 ----
 
+## More Info
+
+Our default chart is built for the use-case of backing up and restoring the Rancher application.
+However, under the hood the Backup Restore Operator is rather flexible extension for backup and restore of Kubernetes resources.
+
+* This operator provides ability to backup and restore Kubernetes applications (metadata) running on any cluster. It accepts a list of resources that need to be backed up for a particular application. It then gathers these resources by querying the Kubernetes API server, packages all the resources to create a tarball file and pushes it to the configured backup storage location. Since it gathers resources by querying the API server, it can back up applications from any type of Kubernetes cluster.
+* The operator preserves the ownerReferences on all resources, hence maintaining dependencies between objects.
```

Comment:
"ownerReferences" should be in backquotes (or courier typeface) as it's a technical term